src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -567,7 +567,13 @@ public void setExtraDockerLabelsString(String extraDockerLabelsString) {
 
     public DockerRegistryEndpoint getRegistry() {
         if (registry == null) {
-            registry = new DockerRegistryEndpoint(null, pullCredentialsId);
+            /* TODO: Use DockerRegistryEndpoint#fromImageName when JENKINS-63243 is fixed */
```

Comment:
Is there a PR for this issue yet?

Reply:
https://github.com/jenkinsci/docker-commons-plugin/pull/88

Reply:
That PR had to wait a long time, but it looks like that PR was merged in on 24 Aug, and went into docker-commons 1.20 onwards.

Could you update this PR to use the new code (i.e. resolve the TODO comment)?